test.py

```diff
@@ -0,0 +1,43 @@
+import marimo
+
+__generated_with = "0.11.19"
+app = marimo.App(width="medium")
+
+
+@app.cell
+def _():
+    import marimo as mo
+
+    mo.raw_cli_args()
+    return (mo,)
+
+
+@app.cell
+def _(mo):
+    b = mo.ui.number(cli_name="test")
```

Comment:
we've actually talked about adding `key`s to UI elements, as it will have other use cases. 

another option that you can do today, why not initialize this with `mo.ui.number(value=mo.cli_args().get("test"))`?